pkg/api/install/install.go

```diff
@@ -7,16 +7,24 @@ import (
 
 	// we have a strong dependency on kube objects for deployments and scale
 	_ "k8s.io/kubernetes/pkg/api/install"
+	_ "k8s.io/kubernetes/pkg/apis/apps/install"
 	_ "k8s.io/kubernetes/pkg/apis/authentication/install"
+	_ "k8s.io/kubernetes/pkg/apis/authorization/install"
 	_ "k8s.io/kubernetes/pkg/apis/autoscaling/install"
 	_ "k8s.io/kubernetes/pkg/apis/batch/install"
+	_ "k8s.io/kubernetes/pkg/apis/certificates/install"
 	_ "k8s.io/kubernetes/pkg/apis/extensions/install"
+	_ "k8s.io/kubernetes/pkg/apis/policy/install"
+	_ "k8s.io/kubernetes/pkg/apis/rbac/install"
+	_ "k8s.io/kubernetes/pkg/apis/settings/install"
+	_ "k8s.io/kubernetes/pkg/apis/storage/install"
 
 	_ "github.com/openshift/origin/pkg/authorization/api/install"
 	_ "github.com/openshift/origin/pkg/build/api/install"
 	_ "github.com/openshift/origin/pkg/cmd/server/api/install"
 	_ "github.com/openshift/origin/pkg/deploy/api/install"
 	_ "github.com/openshift/origin/pkg/image/api/install"
+	_ "github.com/openshift/origin/pkg/kubecompat/apis/extensions/v1beta1"
```

Comment:
Nit, but can we make it more explicit we carry this, and most importantly when it will be dropped. I'd suggest a separate "group" inside imports just for this compatibility stuff. I want this to stand out clearly.